mod-source-record-storage-server/src/main/java/org/folio/services/handlers/match/AbstractMarcMatchEventHandler.java

```diff
@@ -179,7 +179,7 @@ private Filter.ComparisonPartType prepareComparisonPartType(MatchDetail matchDet
   private Future<RecordCollection> processDefaultMatchField(MatchField matchField, List<String> matchedRecordIds, String tenantId) {
     Condition condition = filterRecordByMultipleIds(matchedRecordIds);
```

Comment:
could we create condition in a single line, like: 
filterRecordByMultipleIds(matchedRecordIds).and(filterRecordByState(Record.State.ACTUAL.value()))